Refit/MultipartItem.cs

```diff
@@ -7,12 +7,13 @@ namespace Refit
 {
     public abstract class MultipartItem
     {
-        public MultipartItem(string fileName, string contentType)
+        public MultipartItem(string fileName, string contentType, string name)
```

Comment:
Can you please add another ctor so it's not a breaking change?

Reply:
It's a better idea, I push a new commit for it. thank you.